Make.inc

```diff
@@ -1396,15 +1396,15 @@ clean-$$(abspath $(2)/$(3)):
 ifeq ($(BUILD_OS), WINNT)
 	-cmd //C rmdir $$(call mingw_to_dos,$(2)/$(3),cd $(2) &&)
 else
-	-rm -r $$(abspath $(2)/$(3))
+	-rm -rf $$(abspath $(2)/$(3))
```

Comment:
Maybe then also remove the `-` prefix to these and other `rm` invocations (so that the exit code is no longer ignored)?

Reply:
Done